timescaledb/how-to-guides/connecting/dbeaver.md

```diff
@@ -34,7 +34,9 @@ services.
 1.  Click the `Finish` button to connect to the database server. The server is
     listed in the `Database Navigator` pane.
 
-    <img class="maincontent__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/on_premise_dbeaver.png"alt="Connecting to TimescaleDB using DBeaver"/>
+    <img class="maincontent__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/on_premise_dbeaver.png"
+    alt="Connecting to TimescaleDB using DBeaver"/>
```

Comment:
Curious: is markdownlint doing this? I like it, if so 😁

Reply:
I think it might be Prettier, not markdownlint? Only about 70% sure about this